partiql-eval/Cargo.toml

```diff
@@ -20,8 +20,8 @@ edition.workspace = true
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-partiql-logical = { path = "../partiql-logical" }
-partiql-value = { path = "../partiql-value" }
+partiql-logical = { path = "../partiql-logical", version = "*" }
```

Comment:
Would it be better to set it to the
- current workspace version ("0.2.0") <- would require updating for every release version
- or "*"?

Reply:
Nvm, we cannot use "*" when publishing to crates.io 

>error: failed to publish to registry at https://crates.io
Caused by:
  the remote server responded with an error: wildcard (`*`) dependency constraints are not allowed on crates.io. See https://doc.rust-lang.org/cargo/faq.html#can-libraries-use--as-a-version-for-their-dependencies for more information

Reply:
Will change to use explicit version with wildcard -- e.g. 0.2.*